Themis/Views/Assessment/AssessmentView.swift

```diff
@@ -124,16 +124,6 @@ struct AssessmentView: View {
                 }
                 .foregroundColor(.white)
             }
-            if cvm.currentlySelecting {
-                ToolbarItem(placement: .navigationBarTrailing) {
-                    Button {
-                        cvm.showAddFeedback.toggle()
-                    } label: {
-                        Text("Feedback")
-                    }
-                    .disabled(vm.readOnly)
-                }
-            }
```

Comment:
Note: @AlexExcellence told me we don't need this any more and it was somehow causing build issues, so I just removed it directly.